src/components/HelpResourcesCards/HelpResourcesCards.js

```diff
@@ -0,0 +1,116 @@
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  IconHelpOutline,
+  IconReduceCapacity,
+  IconFlag,
+  IconMap,
+  IconAlternateEmail,
+  CardGroup,
+} from "@trussworks/react-uswds";
+
+export const HelpResourcesCards = () => {
+  return (
+    <aside>
+      <CardGroup>
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader>
+            <h2>Related Resources</h2>
+            <IconReduceCapacity size={4} title="Icon for API Docs" />
+          </CardHeader>
+
+          <CardBody>
+            <p>View resources related to CAMD data</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/related-resources">Related Resources</RouterLink>
+          </CardFooter>
+        </Card>
+
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader align="center">
+            <h2>Release Notes</h2>
+            <IconFlag
+              size={4}
+              title="Icon for other card"
+              className="palette-color-grayscale"
+            />
+          </CardHeader>
+
+          <CardBody>
+            <p>Information on CAM API Updates.</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/release-notes">Release Notes</RouterLink>
+          </CardFooter>
+        </Card>
+
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader align="center">
+            <h2>FAQs</h2>
+            <IconHelpOutline
+              size={4}
+              title="Icon for other card"
+              className="palette-color-grayscale"
+            />
+          </CardHeader>
+
+          <CardBody>
+            <p>Frequently Asked Questions about the CAM API.</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/faqs">FAQs</RouterLink>
+          </CardFooter>
+        </Card>
+
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader align="center">
+            <h2>API Guides</h2>
+            <IconMap
+              size={4}
+              title="Icon for other card"
+              className="palette-color-grayscale"
+            />
+          </CardHeader>
+
+          <CardBody>
+            <p>View guides on how to use CAMD's API.</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/api-guides">API Guides</RouterLink>
+          </CardFooter>
+        </Card>
+
+        <Card gridLayout={{ tablet: { col: 4 } }}>
+          <CardHeader align="center">
+            <h2>Contact Us</h2>
+            <IconAlternateEmail
+              size={4}
+              title="Icon for other card"
+              className="palette-color-grayscale"
+            />
+          </CardHeader>
+
+          <CardBody>
+            <p>Reach out to CAMD for further questions.</p>
+          </CardBody>
+
+          <CardFooter>
+            <RouterLink to="/contact-us">Contact Us</RouterLink>
```

Comment:
So this one will be a Link to https://www.epa.gov/airmarkets/forms/cam-api-contact-us

Reply:
Both should work, but we should be consistent. According to what I've researched, you usually use the javascript route to do some validation before opening the link. We should be consistent, but I'm not sure which we should use. I'm leaning towards the html tag versus the javascript window.open, but I'd like to hear what others think.